tensorrt_llm/_torch/models/modeling_kimi_linear.py

```
@@ -2304,25 +2300,44 @@ def _load_trunk_params(
                 mla_tp_rank = layer.self_attn._mla_tp_rank
                 break
 
+        COL = TensorParallelMode.COLUMN   # shards output rows (dim 0)
+        ROW = TensorParallelMode.ROW      # shards input columns (dim 1)
+        is_kda_by_layer = [getattr(l, "is_kda", False)
+                           for l in self.model.layers]
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)